From 5da31797eefc9fc397582570f430e50153777a94 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:11 -0400
Subject: [redhat] kabi: Adding symbol end_page_writeback (mm/filemap.c)

Message-id: <b5a747cf241b23612ba03eade620a8448af9cc65.1410510773.git.poros@redhat.com>
Patchwork-id: 95984
O-Subject: [PATCH RHEL7.1 071/233] kabi: Adding symbol end_page_writeback (mm/filemap.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    End writeback against a page.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/end_page_writeback
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/end_page_writeback
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/end_page_writeback

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/end_page_writeback b/redhat/kabi/kabi-module/kabi_ppc64/end_page_writeback
new file mode 100644
index 0000000..64beb90
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/end_page_writeback
@@ -0,0 +1,2 @@
+#1-
+0x6033853b end_page_writeback vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/end_page_writeback b/redhat/kabi/kabi-module/kabi_s390x/end_page_writeback
new file mode 100644
index 0000000..7b37d55
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/end_page_writeback
@@ -0,0 +1,2 @@
+#1-
+0x473a42d9 end_page_writeback vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/end_page_writeback b/redhat/kabi/kabi-module/kabi_x86_64/end_page_writeback
new file mode 100644
index 0000000..5b37be1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/end_page_writeback
@@ -0,0 +1,2 @@
+#1-
+0xdbe86ff6 end_page_writeback vmlinux EXPORT_SYMBOL
-- 
1.7.1